js/runner.go

```diff
@@ -435,7 +435,7 @@ func (r *Runner) HandleSummary(ctx context.Context, summary *lib.Summary) (map[s
 func (r *Runner) SetOptions(opts lib.Options) error {
 	r.Bundle.Options = opts
 	r.RPSLimit = nil
-	if rps := opts.RPS; rps.Valid {
+	if rps := opts.RPS; rps.Valid && rps.Int64 > 0 {
```

Comment:
I do wonder if it won't be better to log a warning/info that the rps is <=0 and consequently won't be applied :thinking:

Reply:
As I said in https://github.com/grafana/k6/issues/2613#issuecomment-1192501598, I don't consider zero or negative values as problematic, or worthy of a warning. I think it's usually clear enough when you give `0` or a negative number in such limit-related options that it means "unlimited".

Reply:
:shrug: I would argue we need to warn on `rps` usage so that people just don't use it. But this is separate discussion ;). 

I guess you are write there will be no confusion, and I doubt there are that many users either way :shrug: